certification/internal/engine/engine.go

```diff
@@ -100,7 +107,7 @@ func (c *CraneEngine) ExecuteChecks() error {
 		return fmt.Errorf("%w: %s", errors.ErrExtractingTarball, err)
 	}
 
-	if err := writeCertImage(img); err != nil {
+	if err := writeCertImage(img, tag); err != nil {
```

Comment:
rather than change this signature, why not just create the tag in the writeCertImage func? It makes more sense there, since we aren't using the tag anywhere else in this func.